connect_ext_ppr/models/enums.py

```diff
@@ -0,0 +1,20 @@
+import enum
+
+
+class DeploymentStatusChoices(str, enum.Enum):
+    PENDING = 'pending'
+    PROCESSING = 'processing'
+    SYNCED = 'synced'
+
+
+class MimeTypeChoices(str, enum.Enum):
+    application_json = 'application/json'
```

Comment:
[COD-C] Just an idea - to write names of variants capitalized for unification, might it be convenient?

Reply:
For mime types, media service (that is where I take this), is doing it this way, maybe it not a problem for this Emun to keep this way, and for status capital keys, wdyt?